pkg/cmd/service/service_test.go

```diff
@@ -713,3 +725,81 @@ func TestHTTPServerEnabled(t *testing.T) {
 
 	cancel()
 }
+
+func TestDefaultConfig(t *testing.T) {
```

Comment:
I added this with the hope that it will make sure what's in our `.config-schema.json` file aligns with the actual server defaults.

Reply:
Would it make sense to do this in prod instead of redefining the DefaultConfig?

So DefaultConfig gets read from the schema?

Or on build the schema gets written by the default config? Just in case we forget to add new config options to the test

Reply:
I debated this in https://github.com/openfga/openfga/pull/63#discussion_r903099282

I ended up going with this approach because of the overhead and difficulty of generating a default Go struct from a json-schema, and I felt that the Go struct should be the source of truth and not the other way around (since the Go struct is the definitive entity defining the actual defaults). This approach also avoids having to embed the `.config-schema.json` file in the built container (minor albeit extra dependencies required inside the container). 

Ultimately I could go either way. I'm not opposed to going this approach, and I've already researched what it would take. It just seemed like a lot of overhead for little value 🤷 .